src/server/string_family.cc

```diff
@@ -402,9 +402,9 @@ OpStatus SetCmd::Set(const SetParams& params, string_view key, string_view value
   it->second = std::move(tvalue);
   db_slice.PostUpdate(op_args_.db_cntx.db_index, it, key, false);
 
-  if (params.expire_after_ms) {
-    db_slice.UpdateExpire(op_args_.db_cntx.db_index, it,
-                          params.expire_after_ms + op_args_.db_cntx.time_now_ms);
+  if (params.expire_after_ms && !it->second.HasExpire()) {
```

Comment:
` && !it->second.HasExpire()` no needed, it's a a new value.

Reply:
ok